backstage/packages/backend/src/plugins/scaffolder/actions/provisioner.ts

```diff
@@ -67,13 +71,38 @@ export const provisionNewResourceAction = (config: Config) => {
             type: 'string',
             title: 'serviceOwners',
           },
+          totalBudget: {
+            type: 'number',
+            title: 'totalBudget',
+          },
+          notifyList: {
+            type: 'string',
+            title: 'notifyList',
+          },
         },
       },
     },
 
     async handler(ctx) {
       const requestId = generateRequestId();
       const provisionerConfig = getConfig(config);
+      if (ctx.input.notifyList) {
+        const notifyListArray = ctx.input.notifyList
+          .trim()
+          .split(/,\s*/)
+          .filter(str => str.length);
+
+        ctx.output('notify_list', notifyListArray);
+      }
+
+      if (ctx.input.serviceOwners) {
+        const serviceOwnersArray = ctx.input.serviceOwners
```

Comment:
**Non-blocking**: I'd suggest using an empty string as a default value so we don't need the `if`s here, and to handle the `undefined` value in the output. We can do that in a follow-up PR.